fern/docs.yml

```diff
@@ -76,9 +76,9 @@ js:
   - path: ./dist/output.js
     strategy: beforeInteractive
   - path: assets/heap.js
-    strategy: beforeInteractive
+    strategy: afterInteractive
   - path: assets/dashboard-referral.js
-    strategy: beforeInteractive
+    strategy: afterInteractive
```

Comment:
wait it looks like below you are removing this script entirely? can't this be removed from `docs.yml` too then?